src/ondisk.rs

```diff
@@ -8514,9 +8514,26 @@ pub enum BaudRate {
     _57600Baud = 7,
     #[cfg_attr(feature = "serde", serde(rename = "115200 Baud"))]
     _115200Baud = 8,
-    /// Note: This variant needs a custom AgesaBootloader.
+    /// The variant meaning 230400 Baud only got added with Turin.
+    /// The variant meaning Hacky 3000000 Baud got added with one custom Milan bootloader build.
+    #[cfg_attr(feature = "serde", serde(rename = "230400 Baud"))]
+    #[cfg_attr(feature = "serde", serde(alias = "Hacky 3000000 Baud"))]
```

Comment:
If we're gonna make it a flag day type change, is it worth even keeping this alias? Might as well just update the custom milan configs and make em use the raw value:
```json5
{
	Unknown: {
		entry_id: "Byte",
		// AblSerialBaudRate
		tag: 0xAE46CEA4,
		// 3 MBaud (Milan Fast Spew)
		value: 9
	}
}
```